src/app/shared/components/notification/notification.component.html

```diff
@@ -0,0 +1,22 @@
+<ng-container *ngIf="data.type === notificationTypes.Error">
```

Comment:
ad i can see we have the same template and all the differe`nce is class. so we can use ng class instead of containers

Reply:
and styles of message does not correspond styles of mock-ups